pkg/datapath/linux/requirements.go

```diff
@@ -176,7 +176,12 @@ func CheckMinRequirements() {
 	if !option.Config.DryMode {
 		probeManager := probes.NewProbeManager()
 		if err := probeManager.SystemConfigProbes(); err != nil {
-			log.WithError(err).Warning("BPF system config check: NOT OK.")
+			errMsg := "BPF system config check: NOT OK."
+			if err == probes.ErrKernelConfigNotFound {
```

Comment:
As long as we use the single instance of this error, we should be fine. But more robust approach would be to introduce an error type, and then to check the underlying type of the returned error. Because now we are comparing pointer addrs.

Reply:
Are you suggesting something like this for error checking?
```
swtich err.(type) {
case *probes.ErrKernelConfigNotFound:
    ...
default:
    ...
```

Reply:
Yeah, or even:
```
if _, ok := err.(*probes.ErrKernelConfigNotFound); ok { ... }
```

Just keep in mind, that `type ErrKernelConfigNotFound struct{}` must implement `func Error(string)`.